surface/include/pcl/surface/advancing_front_point_type.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ * Point Cloud Library (PCL) - www.pointclouds.org
+ * Copyright (c) 2017-, Southwest Research Institute
+ *
```

Comment:
I think Sergey already mentioned this somewhere else but you need to add Open Perception here as well.